AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/hearing-details/hearing-details.component.spec.ts

```diff
@@ -142,6 +155,23 @@ describe('HearingDetailsComponent', () => {
         const result = component.getParticipantInfo('123-1234');
         expect(result).toBe('');
     });
+    fdescribe('feature flag turn on and off ', () => {
+        fit('should not show allocated to if work allocation feature flat is off', async () => {
+            launchDarklyServiceSpy.flagChange.next({ 'vho-work-allocation': false });
+            await component.ngOnInit();
+            fixture.detectChanges();
+            const divTohide = fixture.debugElement.query(By.css('hearing-allocated-to'));
+            expect(divTohide).toBeFalsy();
+        });
+
+        it('should  show allocated to if work allocation feature flat is on', async () => {
```

Comment:
double space between should _ _show
Spelling mistake: feature **flag**

Reply:
changed.

Reply:
still a double space betwen should and show